SwiftLeeds/Views/Local/LocalView.swift

```diff
@@ -57,6 +94,34 @@ struct LocalView: View {
             await model.loadData()
         }
     }
+
+    private func locationInfoView(category: Local.LocationCategory, location: Local.Location) -> some View {
```

Comment:
It might be nicer to have this as a full `struct LocationView` which just gets passed these params instead, that way it can live in its own file with its own preview etc?

Reply:
Happy with where this is for now